xAPI-Communication.md

```diff
@@ -1915,7 +1687,7 @@ conformance requirement for the sake of security. In these cases, implementers a
 interoperability implications of their implementation decisions. In any case, the LRS will still need to be configurable such that it
 is able to pass conformance tests. 
 
-While other security concerns are beyond the scope of this specification, the xAPI Community remains dedicated to determining 
+While security concerns are beyond the scope of this specification, the xAPI Community remains dedicated to determining 
 security best practices. This effort has begun at [xAPIsec](https://github.com/xapisec/xapisec). 
```

Comment:
maybe a separate issue, but the last commit to https://github.com/xapisec/xapisec was 3 years ago. Maybe remove this. @blakeplock - would you defend keeping this in?

Reply:
It should be stripped out of 9274. xAPIsec was one of Jason's pet projects and should probably be closed out. While there is some reasonable best-practice info in there, it's best dealt with as elements of a general guidelines "recommended practice" paper for 9274-related security under a yet-to-be-developed 9274.4.2 (all of the RPs will go under .4).

Reply:
Incidentally, there is a chapter on security being written in the ADL-Academy and I expect there will be a few risk IDs and a few mitigating best practices to come out of that work. Will be something to look at to help guide 9274.4.2 which I'm foreseeing as a pretty short paper mostly just defining best practice primarily according to existing best practices outside xAPI.

Reply:
Generally of the opinion that we should keep some encouragement and pointers to security best practices in there. xAPISec was one of those, other best practices for security around web services are fine also IMO.

Reply:
> Generally of the opinion that we should keep some encouragement and pointers to security best practices in there. xAPISec was one of those, other best practices for security around web services are fine also IMO.

But xAPIsec never really came to fruition and isn't maintained. I'd argue we're better off punting and then do the RP report. Alternately, we could do a draft RP (like we did for the xAPI Implementation tech report) on a small scale of a few pages tops and reference it. But even that becomes an issue because we need this 9274 doc to be something that would not need to be fiddled with for several years. Hmm.

Reply:
So, should we just take these lines out completely? or suggest that there will be a best practice guide to look for? I think we don't need to _say_ there will be if we create one, but I can see the argument for saying something as well.

Reply:
https://github.com/bscSCORM/xAPI-Spec/commit/daed4c124221b14772d23d7171703c7f503ac385

Note just a few lines up we say: Security is beyond the 
> Security is beyond the  scope of this document....